.devcontainer/devcontainer.json

```
@@ -30,5 +30,8 @@
     "postCreateCommand": ". /usr/local/share/nvm/nvm.sh; nvm install",
 
     // Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-    "remoteUser": "node"
+    "remoteUser": "node",
+    "features": {
+        "azure-cli": "latest"
```

```suggestion
        "azure-cli": "latest",
        "github-cli: "latest",
        "git-lfs": "latest"
```

I suggest we take the suggestions from @alexvy86. @RishhiB and I were talking about the fact that the GH CLI is not in our codespace config just this morning.